src/EnergyPlus/ReportSizingManager.cc

```diff
@@ -1238,6 +1238,22 @@ namespace ReportSizingManager {
 						ShowContinueError( "SizingString = " + SizingString + ", DataCapacityUsedForSizing = " + TrimSigDigits( DataCapacityUsedForSizing, 1 ) );
 						ShowContinueError( "SizingString = " + SizingString + ", DataFlowUsedForSizing = " + TrimSigDigits( DataFlowUsedForSizing, 1 ) );
 					}
```

Comment:
New function to calculate SAT based on zone temp and not coil inlet temp. Using zone temp represents SAT needed to meet a zone load, using coil inlet temp represents SAT needed to meet a system capacity. Subtle difference.